java/client/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.vitess</groupId>
     <artifactId>vitess-parent</artifactId>
-    <version>12.0.0-SNAPSHOT</version>
+    <version>13.0.0-SNAPSHOT</version>
```

Comment:
aren't these changes already applied by https://github.com/vitessio/vitess/pull/9066?

Reply:
Yeah, not sure why it still shows them in the diff. I can rebase on main, but in the end this should not change either way when merged.

Reply:
rebased on main and they're gone from the diff.